etl/steps/data/garden/AI/2023-06-08/yougov_robots.meta.yml

```diff
@@ -0,0 +1,209 @@
+dataset:
+  title: Robot intelligence survey (YouGov, 2023)
+  description: Biannual tracker on how worried are Americans robot intelligence? Next update is in July. The provided results are weighted to ensure they are representative of the United States population across various demographic factors. The weighting is based on factors such as age, gender, race, education, region, political party affiliation, and income level. This weighting process helps to account for the diversity within the population and ensures that the results reflect a broader representation of the United States as a whole. Each wave comprises responses from 1004 - 1085 US Adults.
```

Comment:
The first sentence seems odd (like in the snapshot). I suppose an "about" is missing.